styles/styles.css

```diff
@@ -423,14 +486,172 @@ body {
         width: calc(var(--col-sm-width) * 6);
     }
 
-    .top {
-        overflow: hidden;
-    }
-
     .wrapper {
         padding: 0 16px 0;
     }
 
+    .wrapper--top {
+        padding-bottom: 40px;
+    }
+
+    .nav {
+        height: 50px;
+        margin-bottom: 40px;
+    }
+
+    .nav__links {
+        display: none;
+    }
+
+    .nav__icons {
+        gap: 16px;
+    }
+
+    .icon__logo {
+        width: 120px;
+        height: 24px;
+    }
+
+    .icon__night {
+        width: 16px;
+        height: 16px;
+    }
+
+    .icon__account {
+        width: 24px;
+        height: 24px;
+    }
+
+    .icon__burger {
+        display: revert;
+        width: 24px;
+        height: 16px;
+    }
+
+    .booking__header--mb {
+        margin-bottom: 30px;
+    }
+
+    .booking__title {
+        font-size: 30px;
+    }
+
+    .booking__labels {
+        display: none;
+    }
+
+    .booking__form {
+        display: grid;
+        height: auto;
+        grid-template-columns: repeat(6, 1fr);
+        grid-template-rows: repeat(4, 1fr);
+        gap: 8px 8px;
+        padding: 8px;
+        background: #F5BD41;
+        border-radius: 0;
+        margin-bottom: 35px;
+    }
+
+    .booking__form--text {
+        font-size: 15px;
+        font-weight: 500;
+    }
+
+    .booking__input::placeholder {
+        font-weight: bold;
+    }
+
+    .booking__dates {
+        border: none;
+        width: 100%;
+        gap: 8px;
+        grid-row: 2;
+        grid-column: span 6;
+    }
+
+    .booking__people {
+        background: #FFFFFF;
+        border-radius: 8px;
+        width: 100%;
+        grid-row: 3;
+        grid-column: span 6;
+    }
+
+    .label__wrap {
+        display: flex;
+        width: 100%;
+        text-align: revert;
+        background-color: #FFFFFF;
+        border-radius: 8px;
+    }
+
+    #city {
+        padding-left: 36px;
+    }
+
+    .label__booking {
+        display: block;
+        order: revert;
+        font-size: 12px;
+        color: #383838;
+    }
+
+    .label__wrap--date {
```

Comment:
As you can see,  your calsses in one block have the same properties, so you can unite it in one universe classe and reuse it